### ISSUE TITLE
Question on Chaoyang dataset noise rate

### ISSUE THREAD

**Issue by gernose** [bupt-ai-cz/HSA-NRL#7]
Hi, thanks for sharing this great project! Since there is no clean labels of training set and no noisy labels of test set, I am wondering the noise rate of Chaoyang dataset. This noise rate of real-world dataset is important to design the synthetic label noise settiing. 

Also, I found the script step1.py defines args.noise_rate = 0.15 for Chaoyang datset. Is this value computed with clean labels, or estimated by other algorithm? 
https://github.com/bupt-ai-cz/HSA-NRL/blob/7375154dfb8787e822a4f7ae1f2bc2d98365a602/step1.py#L66
Thanks~

**Comment by buptcwk** [comment]
It is an estimated noise ratio of the Chaoyang dataset. Since the Chaoyang dataset is constructed in a real-world situation (as described on page 6 of our paper), we do not have the "clean" labels of the training set.
